src/PaymentServer/Server.hs

```diff
@@ -94,5 +94,5 @@ paymentServerApp corsOrigins stripeConfig issuer =
 -- | Create middleware which captures metrics for the payment server app.
 makeMetricsMiddleware :: IO (Application -> Application)
 makeMetricsMiddleware = do
-  meters <- makeMeters paymentServerAPI WithQuantiles
-  return $ monitorServant paymentServerAPI meters
+  meters <- register $ meters paymentServerAPI
```

Comment:
I was ready to be annoyed at servant-prometheus changing in a way that forces us to rely on global mutable state for metric registration.  Then I dug into the prior implementation of `makeMeters` and of course I was thrilled to see that the way it worked was merely by calling `register` itself.  Joy.  At least `meters` is pure and the global state is now readily apparent in our use of the library.  That's a net improvement, I guess.

Reply:
🤷 to me it's all just copy pasta that happens to compile